client/src/components/MultiJurisdictionAudit/Setup/Contests/index.tsx

```diff
@@ -108,22 +108,18 @@ const Contests: React.FC<IProps> = ({
                       /* eslint-disable react/no-array-index-key */
                       <React.Fragment key={i}>
                         {i > 0 && (
-                          /* istanbul ignore next */
                           <FormSection>
                             <hr />
                           </FormSection>
                         )}
                         <FormSection
                           label={`Contest ${
-                            /* istanbul ignore next */
                             values.contests.length > 1 ? i + 1 : ''
                           } Info`}
                           description="Enter the name of the contest that will drive the audit."
```

Comment:
I'm not sure this line of copy makes sense in the case where there are multiple targeted contests

Reply:
@mcchilders Thoughts on this?